### DISCUSSION TITLE
post/running-dotnet-console-apps-aci

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#44] [Announcements]
# post/running-dotnet-console-apps-aci

Mark Heath's Development Blog

https://markheath.net/post/running-dotnet-console-apps-aci

**Comment by DAXaholic** [comment]
You can fix your issue with the variables not visible in your script block by using `$using` (no pun intended :-))

    for ($i = 1; $i -le 3; $i++) {
        Start-Job -ScriptBlock { $using:i }
    }
